docs/security/cloud-infrastructure-security/cloud-infrastructure-security-for-aws.md

```diff
@@ -359,7 +359,7 @@ In this step, you perform the steps needed to deploy the Cloud Infrastructure Se
        <details>
        <summary>CloudQuery</summary>
 
-       In this section, you have the choice to create a [CloudQuery source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cloudquery-source) in Sumo Logic. (This functionality is in [preview](/release-notes-service/2024/05/13/apps/). If you would like to know more about the feature, reach out to your Sumo Logic Account Executive.) 
+       In this section, you have the choice to create a [CloudQuery source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cloudquery-source) in Sumo Logic. 
```

Comment:
To clarify - does this mean user has their choice of any CloudQuery source? Or is this a completely optional step altogether?

Reply:
It's an optional step.

Reply:
Oh ok, thank you for clarifying. I'd maybe edit it like....

```suggestion
       This section demonstrates how you can optionally create a [CloudQuery source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cloudquery-source) in Sumo Logic. 
```

Reply:
Sounds good!